.circleci/config.yml

```diff
@@ -122,7 +122,6 @@ commands:
       - run:
           name: Send job status
           command: |
-            set -x
```

Comment:
Unneeded and noisy

Reply:
(let's keep `set -x` in places where it helps understand the output though, like fuzzing)

Reply:
for now I agree, here, I think this just mangles the logs.